backend/src/index.js

```diff
@@ -1,54 +1,58 @@
-
 // dotenv: access local environmental constants / variables
-require('dotenv').config();
+//I change this part to adapt with O.S system
+require('dotenv/config');
 
 const express = require('express');
 const path = require('path')
 const app = express();
 
 // import mongoose & models from our Schema
-const {mongoose,User,NGO,affinities} = require("./Schema");
+const { mongoose, User, NGO, affinities } = require("./Schema");
 
 // Middleware
 app.use(express.json())
 app.use(express.urlencoded({ extended: false }))
 app.use(express.static(path.join(__dirname, 'src')))
+app.use(require('./middleware/middleware'))
```

Comment:
Rather move this from here, to be used only from the post handler inside the routes.js file. We only want this middleware to be called with the POST handler, not on the rest of our endpoints.

Reply:
About that part you mean like router.post('/users', require('./middleware/middleware'), (req, res) =>{})...........?